lib/src/media_recorder.dart

```diff
@@ -1,8 +1,7 @@
+import 'package:webrtc_interface/src/media_recorder.dart' as _interface;
+import 'package:webrtc_interface/webrtc_interface.dart';
+
 import '../flutter_webrtc.dart';
-import 'interface/enums.dart';
-import 'interface/media_recorder.dart' as _interface;
-import 'interface/media_stream.dart';
-import 'interface/media_stream_track.dart';
 
 class MediaRecorder extends _interface.MediaRecorder {
```

Comment:
I want to remove MediaRecorder from the interface because this interface is completely dependent on the platform implementation and is not a standard in the WebRTC API